lib/PuppeteerSharp/Browser.cs

```diff
@@ -46,7 +45,7 @@ public abstract class Browser : IBrowser
         public abstract bool IsClosed { get; }
 
         /// <inheritdoc/>
-        public IBrowserContext DefaultContext { get; protected set; }
+        public IBrowserContext DefaultContext { get; protected init; }
```

Comment:
FYI: Since `Browser` is `public` this is a breaking change, so you _might_ want to leave out the changes from `set` to `init` from this PR.

Reply:
Yeah...